src/screens/PluginStore/UploadPluginModal.tsx

```
@@ -133,126 +134,73 @@ const UploadPluginModal: React.FC<IUploadPluginModalProps> = ({
-              textAlign: 'center',
-              background: '#fafafa',
-              cursor: 'pointer',
-              transition: 'all 0.2s',
-            }}
-            onClick={handleUploadClick}
-          >
-            <FaUpload
-              style={{ fontSize: 48, color: '#ccc', marginBottom: 16 }}
-            />
-            <div style={{ fontSize: 16, fontWeight: 500, marginBottom: 8 }}>
+          <div className={styles.dropzone} onClick={handleUploadClick}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)